app/src/main/java/org/simple/clinic/activity/TheActivity.kt

```diff
@@ -62,7 +62,7 @@ class TheActivity : AppCompatActivity() {
     }
 
     lifecycle
-        .startWith(Started("TheActivity"))
+        .startWith(Started(javaClass.simpleName))
```

Comment:
Not sure. Take in the `javaClass` alone bound by `T` of type `Activity`?

Reply:
This will get removed when we [remove](https://www.pivotaltracker.com/story/show/169281424) the `UiEvent` interface from the `ActivityLifecycle` events. Should we still do this?

Reply:
Discussed offline: Not necessary